### ISSUE TITLE
Italic p serif

### ISSUE THREAD

**Issue by xsrvmy** [stipub/stixfonts#188]
I have noticed that half of the serif on the tails of p and q have been removed compared to the old version of stix (and thus times). I assume this is a deliberate decision given that Cambria also does this. However it actually looked really odd when I used it in math (may the serif is too long?)

Anyways, are there plans to add a cv that would use times-style serifs on p and q?

**Comment by tiroj** [comment]
Yes, that was a deliberate decision. I prefer single serifs—or no serifs—on descenders in italics, as it conforms better with cursive styling. If people don’t like this, I can restore the left serifs, or provide variants.

**Comment by xsrvmy** [comment]
![image](https://user-images.githubusercontent.com/7594338/106695589-8413e080-65a0-11eb-840f-7d1d460fd5ff.png)

In order: DejaVu, Liberation, Caladea (Google's Cambria), STIX

I guess people who use Word are more used to the current style, but the serif on p should probably be shorter to match the one on q, and people who use LaTeX are more used to having both serifs or no serifs on p and q.

BTW talking about character variants, isn't there an issue with variants not working in subscripts?

**Comment by tiroj** [comment]
`BTW talking about character variants, isn't there an issue with variants not working in subscripts?`

I reordered some of the variant lookups for v2.12, so that they are processed before the ssty substitutions. Will require testing.

**Comment by Firestar-Reimu** [comment]
I am the author of #278 and I prefer Liberation style or no serifs

sorry I didn't notice there are others who asked this question
